editor/src/components/canvas/canvas-strategies/strategies/absolute-resize-bounding-box-strategy.spec.browser2.tsx

```diff
@@ -2943,6 +2946,58 @@ describe('Absolute Resize Strategy Canvas Controls', () => {
     expect(renderResult.renderedDOM.getByTestId('guideline-0').style.display).toEqual('block')
     expect(renderResult.renderedDOM.getByTestId('guideline-1').style.display).toEqual('block')
   })
+
+  it('can resize from near the size label', async () => {
+    const renderResult = await renderTestEditorWithCode(
+      makeTestProjectCodeWithSnippet(`
+        <div style={{ width: '100%', height: '100%' }} data-uid='aaa'>
+          <div
+            style={{ backgroundColor: '#aaaaaa33', position: 'absolute', left: 40, top: 50, width: 200, height: 120 }}
+            data-uid='bbb'
+            data-testid='bbb'
+          />
+        </div>
+      `),
+      'await-first-dom-report',
+    )
+
+    const target = EP.appendNewElementPath(TestScenePath, ['aaa', 'bbb'])
+    const dragDelta = windowPoint({ x: 40, y: -25 })
+
+    await renderResult.dispatch([selectComponents([target], false)], true)
+
+    const canvasControl = getResizeControl(renderResult, EdgePositionBottom)
+    if (canvasControl == null) {
+      throw new Error(`Could not find canvas control.`)
+    }
+
+    const resizeCornerBounds = canvasControl.getBoundingClientRect()
+    const startPoint = windowPoint({
+      x: resizeCornerBounds.x + resizeCornerBounds.width / 2,
+      y: resizeCornerBounds.y + resizeCornerBounds.height - 1,
+    })
+
+    const endPoint = {
+      x: startPoint.x + dragDelta.x,
+      y: startPoint.y + dragDelta.y,
+    }
+
+    await mouseDragFromPointToPoint(canvasControl, startPoint, endPoint, { realMouseDown: true })
```

Comment:
does this test break without your actual fix?

Reply:
yes, it does break